src/pythonic/index.md

```diff
@@ -174,6 +174,9 @@ class Sieve:
         primal.sieve_destroy(self.sieve)
 
     def is_prime(self, n):
+        if n > self.upper_bound():
+            raise IndexError("{} not in upper bound {}"
```

Comment:
Updated the [gist](https://gist.github.com/gurgalex/c1e0355cc212f9ab48fe7eb030a98197)
How does the `# panic` comment sound?
Copied your wording on translating a Rust error to be represented in Python.